RNCalendarReminders.m

```diff
@@ -326,21 +421,28 @@ - (NSArray *)serializeReminders:(NSArray *)reminders
                                                        @"longitude": @(alarm.structuredLocation.geoLocation.coordinate.longitude)
                                                        }}
                                       forKey:@"structuredLocation"];
-                    
+
                 }
                 [alarms addObject:formattedAlarm];
             }
             [formedReminder setValue:alarms forKey:_alarms];
         }
-        
+
         if (reminder.startDateComponents) {
             NSDate *reminderStartDate = [calendar dateFromComponents:reminder.startDateComponents];
             [formedReminder setValue:[dateFormatter stringFromDate:reminderStartDate] forKey:_startDate];
         }
         
+        if (reminder.recurrenceRules) {
```

Comment:
That's a great idea didn't come across **hasRecurreceRules**, but that's totally better.